modules/local-authenticators/pom.xml

```diff
@@ -165,6 +165,28 @@
                             </artifactItems>
                         </configuration>
                     </execution>
+                    <!-- Download magiclink authenticator jar from nexus and copy to the localauthenticator/jar folder-->
```

Comment:
Where does this get copied to dropins or plugins?

Reply:
This gets copied to dropins.

Reply:
This has been moved to plugin with the latest commit.